packages/components/result/__tests__/result.spec.ts

```diff
@@ -15,24 +15,24 @@ describe('Result', () => {
   test('status work', async () => {
     const wrapper = ResultMount()
     expect(wrapper.findComponent(IxIcon).exists()).toBeTruthy()
-    expect(wrapper.findComponent(IxIcon).props().name).toEqual('info-circle')
+    expect(wrapper.findComponent(IxIcon).props().name).toEqual('info-circle-filled')
```

Comment:
```suggestion
   expect(wrapper.find('.ix-icon-info-circle-filled').exists()).toBe(true)
```
断言 icon 的地方，都统一换成这种形式吧。

Reply:
Result测试用例中的断言已修改